### PR TITLE
Delete endpoints & Postman collection

### PR DESCRIPTION
**Pull request by marcauberer**
- Add delete endpoints
- Postman collection for testing

### CODE REVIEW

File: src/main/kotlin/de/livepoll/api/service/PollItemService.kt [hunk 1]
```diff
@@ -1,42 +1,55 @@
...
-                    return item.toDtoOut()
-                } else {
-                    throw Exception("Not authorzied")
+                quizItemRepository.findById(pollItemId).orElseGet {
+                    throw ResponseStatusException(HttpStatus.NOT_FOUND, "This poll item does not exist")
+                }.run {
+                    if (userHasAccess(this)) {
+                        return ResponseEntity.ok().body(this.toDtoOut())
+                    } else {
+                        throw ResponseStatusException(HttpStatus.UNAUTHORIZED, "Not authorzied")
```
Comment (Ordinateur-Hack) [inline src/main/kotlin/de/livepoll/api/service/PollItemService.kt]: typo: not authorized

File: src/main/kotlin/de/livepoll/api/service/PollService.kt [hunk 2]
```diff
@@ -44,7 +50,35 @@ class PollService(
...
+
+    }
+
+    fun deletePoll(id: Int) {
+        pollRepository.findById(id).orElseGet { null }.run {
+            if (this.user.username == SecurityContextHolder.getContext().authentication.name) {
+                try {
+                    pollRepository.deleteById(id)
+                } catch (ex: EmptyResultDataAccessException) {
+                }
```
Comment (Ordinateur-Hack) [inline src/main/kotlin/de/livepoll/api/service/PollService.kt:68]: we should handle this error later on

File: src/main/kotlin/de/livepoll/api/service/PollService.kt [hunk 3]
```diff
@@ -44,7 +50,35 @@ class PollService(
...
+                try {
+                    multipleChoiceItemRepository.deleteById(id)
+                } catch (ex: EmptyResultDataAccessException) {
+                }
+                try {
+                    quizItemRepository.deleteById(id)
+                } catch (ex: EmptyResultDataAccessException) {
```
Comment (Ordinateur-Hack) [inline src/main/kotlin/de/livepoll/api/service/PollService.kt:75]: we should handle this error later on

File: src/main/kotlin/de/livepoll/api/service/PollItemService.kt (hunk 1 above)
Comment (marcauberer) [reply src/main/kotlin/de/livepoll/api/service/PollItemService.kt]: Should be fine now

File: src/main/kotlin/de/livepoll/api/service/PollService.kt (hunk 2 above)
Comment (marcauberer) [reply src/main/kotlin/de/livepoll/api/service/PollService.kt:68]: Would you like to fix that real quick?

File: src/main/kotlin/de/livepoll/api/service/PollService.kt (hunk 3 above)
Comment (marcauberer) [reply src/main/kotlin/de/livepoll/api/service/PollService.kt:75]: Would you like to fix that real quick?

**Pull request merged**